src/js/options.js

```diff
@@ -10,14 +10,17 @@ import Notiflix from "notiflix";
 import beautify from "js-beautify";
 import { DateTime } from "luxon";
 import { Sortable } from "sortablejs";
+import BSN from "bootstrap.native/dist/bootstrap-native.esm.min.js";
```

Comment:
_I think_ you could just do `import BSN from "boostrap.native"` (source: https://github.com/thednp/bootstrap.native/wiki/How-to-use#requirejs-commonjs)

Reply:
Yeah, I tried that before too but got this error `node_modules/boostrap.native doesn't exist`